docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Vote Account Management
+---
+
+This page describes how to set up an on-chain _vote account_.   Creating a vote
+account is needed if you plan to run a validator node on Solana.
+
+All settings for a vote account can be configured at the time the account is
+created, or they can be set after the account is created while the corresponding
+validator is running.
+
+### Vote Account Address
+
+A vote account is created at an address that is either the public key of a
+keypair file, or at a derived address based on a keypair file's public key and
+a seed string.
+
+The address of a vote account does not need to have a stored corresponding
+private key, because the address (although it may be a valid public key) is never
+needed to sign any transactions, but is just used to look up the account
+information.
+
+When someone wants to [delegate tokens in a stake account](../staking.md),
+the delegation command is pointed at the vote account address of the validator
+to whom the token-holder wants to delegate.
+
+### Validator Identity
+
+The _validator identity_ is a system account that is used to pay for all the
+vote transaction fees submitted by the vote account.
+Because the vote account is expected to vote on most valid blocks it receives,
+the validator identity account is frequently
+(potentially multiple times per second) signing transactions and
+paying fees.  For this reason the validator identity account's keypair must be
+stored as a "hot wallet" in a keypair file on the same system the validator
+process is running.
+
+Because a hot wallet is generally less secure than an offline or "cold" wallet,
+we only recommend storing enough SOL on the identity account to cover voting fees
```

Comment:
Avoid making recommendations. Instead, state either the benefits of taking some action and/or the consequences of not taking it.